<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>公元纪年-干支纪年转换器</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-Heavenly-stems-and-Earthly-Branches-converter-Desktop_Qt_5_5_1_GCC_32bit-Debug/gan.jpg</normaloff>../build-Heavenly-stems-and-Earthly-Branches-converter-Desktop_Qt_5_5_1_GCC_32bit-Debug/gan.jpg</iconset>
  </property>
  <widget class="QLabel" name="Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>输入公元年</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>干支年</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LineEdit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>转换</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="Label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>---</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
